thorlcr/slave/slavmain.cpp

```diff
@@ -1010,13 +969,14 @@ class CKJService : public CSimpleInterfaceOf<IKJService>, implements IThreaded,
             return service.createProcessor();
         }
     };
-    SimpleHashTableOf<CActivityContext, unsigned> activityContextsHT;
-    CKeyLookupContextHT keyLookupContextsHT; // non-owning HT of CLookupContexts
-    CKMContextHT cachedKMs; // NB: HT by {actId, fname, crc} links IKeyManager's within containers
-    OwningSimpleHashTableOf<CKMContainer, unsigned> cachedKMsByHandle; // HT by { handle }, links IKeyManager's within containers.
-    OwningSimpleHashTableOf<CFetchContext, FetchKey> fetchContextsHT;
-    std::unordered_map<unsigned, CFetchContext *> fetchContextsByHandleHT;
+    std::unordered_map<unsigned, CActivityContext *> activityContextsHT;
+    std::unordered_map<CLookupKey, CKeyLookupContext *, CLookupKey> keyLookupContextsHT;
+    std::unordered_map<CLookupKey, Owned<CKMKeyEntry>, CLookupKey> cachedKMs;
+    std::unordered_map<CFetchKey, Owned<CFCKeyEntry>, CFetchKey> cachedFetchContexts;
+    std::unordered_map<unsigned, Owned<CKMContainer>> activeKManagersByHandle;
+    std::unordered_map<unsigned, Owned<CFetchContext>> activeFetchContextsByHandle;
```

Comment:
Note to reviewer: the change to use std::unordered_map's here, isn't directly related, but I would have need yet more boiler plate code for SuperHashTable's as a result of this change, so standardizing on STL here made the code cleaner.

NB: I tested for performance differences (none/tiny -/+ variance) and leaks with valrgrind.